packages/client-sdk-nodejs/src/internal/pubsub-client.ts

```diff
@@ -63,10 +58,21 @@ export class PubsubClient extends AbstractPubsubClient<ServiceError> {
     const grpcConfig = new StaticGrpcConfiguration({
       deadlineMillis: this.unaryRequestTimeoutMs,
       maxSessionMemoryMb: PubsubClient.DEFAULT_MAX_SESSION_MEMORY_MB,
+      keepAlivePermitWithoutCalls: 1,
+      keepAliveTimeMs: 5000,
+      keepAliveTimeoutMs: 1000,
```

Comment:
this should not be in here. this is what i was working on when i got distracted by the confusing variables... will remove this